var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -24,24 +24,47 @@
 ##############################################################################
 from spack import *
 
-
-class Scr(Package):
+class Scr(CMakePackage):
     """SCR caches checkpoint data in storage on the compute nodes of a
        Linux cluster to provide a fast, scalable checkpoint/restart
        capability for MPI codes"""
 
-    homepage = "https://computation.llnl.gov/project/scr/"
+    homepage = "http://computation.llnl.gov/projects/scalable-checkpoint-restart-for-mpi"
+
+    ## NOTE: scr-v1.1.8 is built with autotools and is not properly build here.
+    ## scr-v1.1.8 will be deprecated with the upcoming release of v1.2.0
+    # url      = "https://github.com/LLNL/scr/releases/download/v1.1.8/scr-1.1.8.tar.gz"
+    # version('1.1.8', '6a0f11ad18e27fcfc00a271ff587b06e')
```

Comment:
Is it worth preserving old versions (i.e. are there going to be codes that depend on them?)  If so, it might be useful to look at `dyninst`, which is an example of a package with two build systems.  Only disadvantage is that it extends `Package`, which lacks many of the nice phases that `AutotoolsPackage` and `CMakePackage` provideI.  I don't think anyone's tried to do this with multiple inheritance, though it was discussed with @alalazo [here](https://github.com/LLNL/spack/issues/3642#issuecomment-290725251) in #3642.